app/population/fields/bedwatch.js

```diff
@@ -0,0 +1,16 @@
+const bedwatch = {
+  validate: ['required', 'numeric'],
+  component: 'govukInput',
+  label: {
+    text: 'fields::bedwatch.label',
+    classes: 'govuk-label--s',
+  },
+  id: 'bedwatch',
+  name: 'bedwatch',
+  classes: 'govuk-input--width-5',
+  autocomplete: 'off',
```

Comment:
A lot of these fields seem to share some similar config. You might be able to reduce the duplication by creating a [template](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/app/move/fields/common.date.js) and [extending it](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/07b8b4dc197b790a985f63944b08de81eb5b1087/app/move/fields/date-to.js#L6) for each field.

Reply:
Refactored to use a construction function as that created less duplication than extending it for each field